tokens/src/utilities/color.json

```diff
@@ -8,5 +8,1051 @@
       },
       "utilityFunctionsToApply": ["bgVariant", "textEmphasisVariant", "borderColor"]
     }
-  ]
-}
\ No newline at end of file
+  ],
+  "color": {
+    "focus": {
```

Comment:
IMO the name should be more generic than `focus`, however when I wrote that I didn't have more ideas, 
probably `action` is better and when someone needs a specific action that could be added like this

```
{
  "color": {
    "action":  {
        "general": {...},
        "focus": {...},
        "hover": {...},
....
      }
  }
}
```
where `general` would be this case, that is used in `focus` and `hover`

Reply:
@andrey-canon I think your proposed approach above is reasonable! 

nit: just personal preference, but perhaps `default` in favor of `general`? 🤔 

Also, I'm wondering if it'd be worthwhile to "namespace" these tokens under a "utilities" property, e.g.:

```
{
  "color": {
      "utilities": {
          "action":  {
              "default": {...},
              "focus": {...},
              "hover": {...}
          }
      }
  }
}
```

such that the resulting CSS variables are perhaps more like:

```css
var(--pgn-color-utilities-action-default)
```

Reply:
@adamstankiewicz I applied in this [commit](https://github.com/openedx/paragon/pull/2022/commits/b6b7c0aef6c925931678ae74b918405bb73920ea) the first approach  and this [commit ](https://github.com/openedx/paragon/pull/2022/commits/e1ea4bdc0ecff8467f1e689e5632b690ea15a6cf) contains the utilities approach however I'm not sure about adding the namespace since that makes me feels those variables will only be used in the utility-classes file, right now is true, but probably, those will be used in other places in the future, so I think a general name will fit better

Reply:
@andrey-canon Totally a valid concern! That said, because these tokens are already defined specifically in the `utilities` folder, it would indicate that they'll only be used relevant to utility classes. If these new tokens are more general purpose than just the utility classes, then perhaps they should be created outside of the `utilities` folder in favor of `tokens/global/color.json`?

Reply:
@adamstankiewicz I agree with moving that to global folder, in that way, the variable scope is not limited

Reply:
+1 sounds good here.

Reply:
applied in this [commit](https://github.com/openedx/paragon/pull/2022/commits/a5fe03d399e1e4ba4aeac05f7778bd73c3e40752)